src/fontstash.h

```diff
@@ -353,7 +387,12 @@ struct FONSglyph
 	int next;
 	short size, blur;
 	short x0,y0,x1,y1;
-	short xadv,xoff,yoff;
+	int oversampleV, oversampleH;
+
+	// @@ Using shorts and mul/div by 10 is probably overkill at 
+	//    this point, unless we REALLY need to save that memory?
+
+	float xadv,xoff,yoff,xoff2,yoff2;
```

Comment:
If oversampleV/H are `char` alignment should be fine, but yeah better to move them.